doc/source/development/contributing_codebase.rst

```diff
@@ -421,12 +421,6 @@ This is an example of a green build.
 
 .. image:: ../_static/ci.png
 
-.. note::
-
-   Each time you push to *your* fork, a *new* run of the tests will be triggered on the CI.
-   You can enable the auto-cancel feature, which removes any non-currently-running tests for that same pull-request, for
-   `Travis-CI here <https://docs.travis-ci.com/user/customizing-the-build/#Building-only-the-latest-commit>`__.
-
```

Comment:
We don't have auto-cancel enabled for GitHub Actions yet. It can be done with the [concurrency feature](https://docs.github.com/en/actions/reference/workflow-syntax-for-github-actions#concurrency).

Reply:
cool should prob add the auto cancel feature